monkey/monkey_island/cc/services/authentication_service/otp_generator.py

```diff
@@ -0,0 +1,16 @@
+from .authentication_facade import AuthenticationFacade
+
+
+class OTPGenerator:
```

Comment:
It seems minor, but can we create `IOTPGenerator` and do `class AuthenticationServiceOTPGenerator(IOTPGenerator)`? It's small, but I'd rather not have other components outside `AuthenticationService` depend on a concrete implementation. `IOTPGenerator` and `AuthenticationServiceOTPGenerator` should be exported in `authentication_service/__init__.py`.

Other names for `AuthenticationServiceOTPGenerator` are welcome.